### DISCUSSION TITLE
Upcoming features

### DISCUSSION THREAD

**Discussion by phplime** [phplime/Qrexorder#2] [Polls]
Please give your vote on which one you need most.

**Comment by sparrowaves** [comment]
Multi-Store Option for restaurants

**Reply by amrhelw** [reply]
+1

**Reply by esmhos** [reply]
important

**Reply by mazenelmasry** [reply]
important

**Reply by stevealanIT** [reply]
Very very important

**Comment by SaidSantiago** [comment]
Please add the option to modify the code to match the correct country code. This is necessary for WhatsApp notifications to work properly. Currently, the prefix for Mexico is set as +52, but it should be +521.

By updating the code to reflect the correct country code, WhatsApp notifications will be able to function correctly.

**Reply by phplime** [reply]
it is already added in version 3.1.9
![Uploading SCR-20240317-rvoq.png…]()

**Reply by SaidSantiago** [reply]
Yeah my bad amazing work thank you so much

**Comment by SaidSantiago** [comment]
Hello, I would like to suggest the following features: it would be great if the point of sale deducted the ounces used to prepare a drink from the inventory. Additionally, it would be helpful if the point of sale system provided a partial shift closure for changing personnel, including the ability to record the starting amount of cash for the cashier on the main page. It would also be beneficial if the system automatically deducted payment to suppliers. Furthermore, it would be incredible to have a system log that registers any product deletions, showing the activity in the superadmin. Lastly, having a system for extending credit, where the deducted amount is reflected as a negative until it is paid, would be a valuable addition.

**Comment by stevealanIT** [comment]
KOT print , should include only items , quantity and basic order details. 
For every new items added a separate KOT to be generated

Deleted items summary instaff account. 

Cancelled orders summary in staff account.
